db/migrate/20190219194426_add_report_status_info_to_potential_payments.rb

```diff
@@ -0,0 +1,8 @@
+class AddReportStatusInfoToPotentialPayments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :potential_payments, :uphold_status_was, :string
```

Comment:
IMO you should drop `_was` in all these columns. It's fairly self explanatory it's a snapshot. 

@corymcdonald @dlipeles do you agree?

Reply:
aye 👍

Reply:
thanks guys, will update

Reply:
fb4c7d7